### PR TITLE
Fix typo and missing parameter to logp.Warn() in libbeat/common/event.go

### PR DESCRIPTION
**Pull request by zstyblik**
Commits do fix a typo and missing parameter to logp.Warn() in `libbeat/common/event.go`.

### CODE REVIEW

**Comment by zstyblik** [comment]
I will finish singing CLA later today.

**Comment by ruflin** [comment]
LGTM.

@zstyblik Please comment again as soon as you have signed the CLA so it checks it again.

**Comment by zstyblik** [comment]
CLA completed.

**Comment by zstyblik** [comment]
I see. I've committed with work email, not with one associated with GH account. Grh!

**Comment by zstyblik** [comment]
Pushed again and signed CLA again.

**Comment by zstyblik** [comment]
@ruflin, please, any ideas why CLA shows as not signed here? Thanks.

File: libbeat/common/event.go [hunk 1]
```diff
@@ -64,7 +64,7 @@ func ConvertToGenericEvent(v MapStr) MapStr {
 			case reflect.Map:
 				anothermap, err := MarshallUnmarshall(value)
 				if err != nil {
-					logp.Warn("fail to marschall & unmarshall map (%v): key=%v value=%#v",
+					logp.Warn("fail to marshall & unmarshall map (%v): key=%v value=%#v",
```
Comment (andrewkroh) [inline libbeat/common/event.go]: This log line has more parameters than arguments.

Comment (zstyblik) [inline libbeat/common/event.go]: @andrewkroh yes, it does(https://github.com/elastic/beats/issues/1380).

**Pull request merged**